pgtrigger/__init__.py

```diff
@@ -27,7 +29,10 @@
 from pgtrigger.core import Update
 from pgtrigger.core import UpdateOf
 
-default_app_config = 'pgtrigger.apps.PGTriggerConfig'
+if django.VERSION < (3, 2):
+    default_app_config = 'pgtrigger.apps.PGTriggerConfig'
```

Comment:
The default_app_config is no longer used as of django 3.2. Fixing this now to remove the warning that django emits